docs/tutorials/install.md

```diff
@@ -4,6 +4,20 @@ order: 3
 
 # Install CometBFT
 
+## From Go Install
+
+Install CometBFT with the following command:
+
+```sh
+go install github.com/cometbft/cometbft/cmd/cometbft@latest
+```
+
+Otherwise, you can install a specific version of CometBFT with the following command:
+
+```sh
+go install github.com/cometbft/cometbft/cmd/cometbft@v0.38.5
```

Comment:
Thanks @dragonkid. Another useful command could be `go install github.com/cometbft/cometbft/cmd/cometbft@v0.38` without specifying the patch. Usually we refer to releases as MAJOR.MINOR, like CometBFT v0.34, v0.37 and v0.38, so if a user wants one of these releases and they are not concerned with a specific patch level but just the latest one in that line, if they don't specify the patch, the `go` command will take care of that. Could you please add that information ?